### ISSUE TITLE
[CODE IMPROVEMENT] Probabilities in causal classification modeling

### ISSUE THREAD

**Issue by binga** [h2oai/h2o-llmstudio#470]
### 🔧 Proposed code refactoring

Currently, the classification task outputs only the class values but not probabilities. It'd be nice if it also outputs class probabilities.

### Motivation

The class probabilities are useful for various downstream tasks. One example is the h2oai competition on Kaggle - https://www.kaggle.com/competitions/h2oai-predict-the-llm.

**Comment by psinger** [comment]
Hi @binga - there is a separate pickle file in the experiment folder containing the logits: `validation_raw_predictions.pkl`

Can be read via:
```
exp_name = "exp-v0"
p = pd.read_pickle(f"~/h2o-llmstudio/output/user/{exp_name}/validation_raw_predictions.pkl")
```

**Comment by binga** [comment]
This helps. Thank you for the super fast response. :)
Feel free to close this issue if there is no plan to add this to the CSV file.
